tools/export/makefile/__init__.py

```diff
@@ -272,11 +272,10 @@ def generate(self):
         if self.resources.linker_script:
             sct_file = self.resources.get_file_refs(FileType.LD_SCRIPT)[-1]
             new_script = self.toolchain.correct_scatter_shebang(
-                sct_file.path, join("..", dirname(sct_file.name)))
+                            sct_file, dirname(sct_file.name))
```

Comment:
This indentation looks off. It looks like it's 12 spaces when it should be 4.

Reply:
fixed